### PR TITLE
Dev python/tikzplotlib: Version bump 0.9.1

### PR DESCRIPTION
**Pull request by AndrewAmmerlaan**
Renamed from matplotlib2tikz
Added py3_7
Fixed IUSE="test"
Added IUSE="doc"

Bug: https://bugs.gentoo.org/695716
Package-Manager: Portage-2.3.87, Repoman-2.3.20
Signed-off-by: Andrew Ammerlaan <andrewammerlaan@riseup.net>

### CODE REVIEW

File: dev-python/tikzplotlib/tikzplotlib-0.9.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		dev-python/pandas[${PYTHON_USEDEP}]
+		sci-libs/scipy[${PYTHON_USEDEP}]
+	)"
+
+distutils_enable_tests pytest
+distutils_enable_sphinx doc dev-python/mock
+
+python_prepare_all() {
+	# this tests fails: tikz error 
+	# cannot mix dimenstions and dimenstionless values in an ellipse
```
Comment (equaeghe) [inline dev-python/tikzplotlib/tikzplotlib-0.9.1.ebuild]: minor: dimenstion → dimension, dimenstionless → dimensionless

Comment (AndrewAmmerlaan) [reply dev-python/tikzplotlib/tikzplotlib-0.9.1.ebuild]: Oops, thank you for catching that
I was doing this on my laptop through ssh and I was using a cli based editor without spell checking :(

**Comment by AndrewAmmerlaan** [comment]
@heroxbd this is a version bump of matplotlib2tikz (as requested in https://bugs.gentoo.org/695716), it got renamed to tikzplotlib along the way. I'm afraid I'm not quite familiar with the procedure for renaming packages. What I've done here, is introduced the version bumped tikzplotlib in commit one, and then removed the old matplotlib2tikz in commit two. Please let me know if this is the correct way of renaming a package

**Comment by heroxbd** [comment]
Andrew Ammerlaan <notifications@github.com> writes:

> @heroxbd this is a version bump of matplotlib2tikz (as requested in
> https://bugs.gentoo.org/695716), it got renamed to tikzplotlib along
> the way. I'm afraid I'm not quite familiar with the procedure for
> renaming packages. What I've done here, is introduced the version
> bumped tikzplotlib in commit one, and then removed the old
> matplotlib2tikz in commit two. Please let me know if this is the
> correct way of renaming a package

Thank you Andrew!

Renames are defined in `profiles/updates` of the gentoo repository.

Benda

**Comment by AndrewAmmerlaan** [comment]
> Thank you Andrew! 
>
> Renames are defined in `profiles/updates` of the gentoo repository. 
>
> Benda
 
Thank you @heroxbd :)
I have added a third commit here that adds `move dev-python/matplotlib2tikz dev-python/tikzplotlib` to `1Q-2020`. Should I also squash these commits, or should they be separate in this case?

**Comment by heroxbd** [comment]
> > Thank you Andrew!
> > Renames are defined in `profiles/updates` of the gentoo repository.
> > Benda
> 
> Thank you @heroxbd :)
> I have added a third commit here that adds `move dev-python/matplotlib2tikz dev-python/tikzplotlib` to `1Q-2020`. Should I also squash these commits, or should they be separate in this case?

That's great!  I prefer squashing them, because they function as a whole.

**Comment by AndrewAmmerlaan** [comment]
> That's great! I prefer squashing them, because they function as a whole.

Alright, I squashed them and pushed the new single commit :)

**Comment by AndrewAmmerlaan** [comment]
Awesome! Thank you :smile:

**Comment by heroxbd** [comment]
Andrew Ammerlaan <notifications@github.com> writes:

> Awesome! Thank you 😄

No problem.  Thank you for your work!

**Pull request closed without merging**